src/components/PlaybackControls/index.tsx

```diff
@@ -100,16 +100,11 @@ const PlayBackControls = ({
         }
     };
 
-    // Determine the width of the input box
+    // Determine the width of the input box based on a likely max number of characters
     const getTimeInputWidth = (): string => {
-        // If lastFrameTime is 15 and step size is 0.25 then 15.25 is probably going to have
-        // the max number of characters for this trajectory
-        const refTimeValue =
-            displayTimes.roundedLastFrameTime + displayTimes.roundedTimeStep;
-        const maxNumChars = refTimeValue.toString().length;
         // If maxNumChars is 5 then the input box width will be 6 character widths long
         // (+ 1 is arbitrary padding)
-        return maxNumChars + 1 + "ch";
+        return displayTimes.maxNumChars + 1 + "ch";
```

Comment:
this might be less ambiguous as a formatted string `${(displayTimes.maxNumChars+1)}ch` or maybe just putting parens around the numbers...